.github/workflows/integration.yml

```
@@ -10,12 +10,7 @@ jobs:
     strategy:
       matrix:
         os: [Ubuntu, MacOS, Windows]
-        python-version: [3.7, 3.8, 3.9]
-        experimental: [false]
-        include:
-          - os: Ubuntu
-            python-version: "3.10.0-alpha - 3.10.0"
-            experimental: true
+        python-version: [3.7, 3.8, 3.9, "3.10"]
```

seems everything is passing but let's keep `experimental` as it's referenced above:
```suggestion
        python-version: [3.7, 3.8, 3.9, "3.10"]
        experimental: [false]
```

---

I think we should remove `continue-on-error: ${{ matrix.experimental }}` or keep it as is.

The reason why is python 3.10 is stable now and considered officially supported in poetry.

I think experimental would be for either or both of these conditions:
- Alpha python release, e.g. 3.11 alpha
- python release unsupported by poetry, e.g. if 3.10 wasn't considered ready for poetry despite python release 3.10.0 officially
- Bootstrapping poetry trunk, e.g. `bootstrap-args: "--git https://github.com/python-poetry/poetry.git"` (but `tox.ini` doesn't support that and it may be best to defer that to a separate PR)

---

agreed, let's remove `continue-on-error: ${{ matrix.experimental }}` for now. 

We'll prob add it back to test 3.11 alpha/beta in the future like you said, but we can handle that then

---

[c3bc556](https://github.com/python-poetry/poetry-core/pull/265/commits/c3bc556a0e7b683553f3a824a13077fcb644322e)